### PR TITLE
Reordering of UG and correcting spelling error in DG

### LINKED ISSUE #134: Re-ordering of table of contents for clearer distinction between main features
Could further order the commands into its categories as mentioned in the main feature

### PR DESCRIPTION
**Pull request by Johnson-Yee**
Regarding #55 
Fixes #134

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ enhancements.
   * [3.9. Bus at bus stop finder (`/bus` Feature)](#39-bus-at-bus-stop-finder-bus-feature)
   * [3.10. Performing similarity checks](#310-performing-similarity-checks)
   * [3.11. Displaying most searched bus stop](#311-displaying-most-searched-bus-stop-on-start-up)
-    + [3.11.1 Resetting all search frequencies](#3111-resetting-search-frequencies)
-- [4. Appendix A: Product scope](#4-appendix-a-product-scope)
+    + [3.11.1 Resetting all search frequencies](#3111-resetting-search-frequencies-of-bus-stops-reset-feature)
+- [4. Appendix I: Requirements](#4-appendix-i-requirements)
```
Comment (wamikamalik) [inline docs/DeveloperGuide.md]:
```suggestion
    + [3.11.1 Resetting all search frequencies](#3111-resetting-search-frequencies-of-bus-stops-reset-feature)
- [4. Appendix A: Requirements](#4-appendix-a-requirements)
```

Was this when I committed, I don't know how it got changed. Can you please change the corresponding header as well.

Comment (wamikamalik) [inline docs/DeveloperGuide.md]: Remember to add 3.12!

Comment (Johnson-Yee) [reply docs/DeveloperGuide.md]: Alright!

Comment (Johnson-Yee) [reply docs/DeveloperGuide.md]: OOo I get what you mean!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -26,8 +26,9 @@ enhancements.
   * [3.9. Bus at bus stop finder (`/bus` Feature)](#39-bus-at-bus-stop-finder-bus-feature)
   * [3.10. Performing similarity checks](#310-performing-similarity-checks)
   * [3.11. Displaying most searched bus stop](#311-displaying-most-searched-bus-stop-on-start-up)
-    + [3.11.1 Resetting all search frequencies](#3111-resetting-search-frequencies)
-- [4. Appendix A: Product scope](#4-appendix-a-product-scope)
+    + [3.11.1 Resetting all search frequencies](#3111-resetting-search-frequencies-of-bus-stops-reset-feature)
+  * [3.12. Removing specific delete command (`/deletefav` Feature)](#312-removing-specific-delete-command-deletefav-feature)
+- [4. Appendix A: Requirements](#4-appendix-a-requirements)
```
Comment (wamikamalik) [inline docs/DeveloperGuide.md]:
```suggestion
- [4. Appendix A: Product Scope](#4-appendix-a-product-scope)
```
Sorry! I just saw the CS2113T website and it says this.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -443,7 +444,7 @@ index has been deleted.
 The following sequence diagram illustrates the steps taken by the program when the user calls the `/deletefav` command.
 ![Sequence ](DG_Diagrams/DeleteFavCommand/DeleteFavSeq.png)
 
-## 4. Appendix I: Requirements
+## 4. Appendix A: Requirements
```
Comment (wamikamalik) [inline docs/DeveloperGuide.md]:
```suggestion
## 4. Appendix A: Product Scope
```

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Johnson-Yee) [reply docs/DeveloperGuide.md]: No worries! Changed!

**Review by wamikamalik — APPROVED** [review]
LGTM!

**Pull request merged**